app/src/org/commcare/preferences/HiddenPreferences.java

```diff
@@ -296,4 +297,18 @@ public static String getLastKnownCczLocation() {
         return PreferenceManager.getDefaultSharedPreferences(CommCareApplication.instance())
                 .getString(LAST_KNOWN_CCZ_LOCATION, null);
     }
+
+    public static void updateLastUploadAttemptTime() {
```

Comment:
Nitpick: everywhere else this is "Upload and Sync attempt," this one only mentions upload